addons/dogtags/functions/fnc_getDogtagItem.sqf

```diff
@@ -33,7 +33,8 @@ private _item = format ["ACE_dogtag_%1", _nextID];
 _allDogtags pushBack _item;
 _allDogtagDatas pushBack _dogTagData;
 
-missionNamespace setVariable [QGVAR(allDogtags), _allDogtags];
-missionNamespace setVariable [QGVAR(allDogtagDatas), _allDogtagDatas];
+// Broadcast data, so that the arsenal can use it client side
+missionNamespace setVariable [QGVAR(allDogtags), _allDogtags, true];
+missionNamespace setVariable [QGVAR(allDogtagDatas), _allDogtagDatas, true];
```

Comment:
this could grow to be quite large
and the whole array will be resynced on each new entry

hate to say, but maybe we should just remove the arsenal thing?

Reply:
> the whole array will be resynced on each new entry

In that case `remoteExec` a fnc with the new entry.

Reply:
I went ahead with my proposition. If this is still deemed too performance intensive, I'll drop it altogether.